plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -75,6 +75,8 @@ synapticsmst_device_kind_to_string (SynapticsMSTDeviceKind kind)
 const gchar *
 synapticsmst_device_board_id_to_string (SynapticsMSTDeviceBoardID board_id)
 {
+	if (board_id == SYNAPTICSMST_DEVICE_BOARDID_DELL_FUTURE)
+		return "Dell future device";
```

Comment:
I don't think this is a very good label. I think something like "Dell Generic SynapticsMST Device" is much better, and less confusing for anyone using this version of fwupd and brand new hardware.

Reply:
Sounds good, i'll change it around.  The most important part is that it doesn't bail for being an unknown type.

Reply:
I wonder if we should use the quirk files for this kind of thing? It means we can support "new" hardware on old RHEL releases just dumping a text file in a known location, rather than backporting a new binary plugin. In the meantime LGTM.

Reply:
Hmm.  Conceptually I like that idea.  When this device is further along I'll see if that is something feasible.